README.md

```diff
@@ -174,6 +174,19 @@ curl --header "Authorization: Token token=GvmkAW231VxGHkYxyppr2QQsi1D7PStqeiJXyy
 curl --header "Authorization: Token token=GvmkAW231VxGHkYxyppr2QQsi1D7PStqeiJXyyja" localhost:3001/v0/profile
 ```
 
+## Development Setup ( Docker )
```

Comment:
Gotcha.  So maybe directly under our [Develop Setup](https://github.com/department-of-veterans-affairs/vets-api#developer-setup) you could put something like:
```
As an alternative, you may use [Docker to setup Postgres, Redis, & specific version of Ruby](https://github.com/department-of-veterans-affairs/vets-api/blob/2b9ba822f0e1aaf0fbb7882574ea69e80fd53611/README.md#development-setup--docker-)
```